cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -112,7 +112,7 @@ func TestAddFlags(t *testing.T) {
 		},
 		Admission: &apiserveroptions.AdmissionOptions{
 			RecommendedPluginOrder: []string{"NamespaceLifecycle", "Initializers", "MutatingAdmissionWebhook", "ValidatingAdmissionWebhook"},
-			DefaultOffPlugins:      []string{"Initializers", "MutatingAdmissionWebhook", "ValidatingAdmissionWebhook"},
+			DefaultEnabledPlugins:  []string{"NamespaceLifecycle"},
 			PluginNames:            []string{"AlwaysDeny"},
```

Comment:
This is the list of manually enabled plugins. We also need the inverse: manually disabled plugins. Otherwise, you can never disable a plugin in `DefaultEnabledPlugins`.

Reply:
DefaultEnabledPlugins will not be exported as flag. So I guess we need `--disable-admission-plugin`.

About the need: DefaultEnabledPlugins will include 90% of the available plugins in kube-apiserver. So users will want to disable some.

Reply:
If so, user must `--disable-admission-plugin`. which may be a burden for cluster admins?

Reply:
the normal case is that the admin leaves all on. If he switches some off or on, he has very good reasons (hopefully). So an addition arg is fine.

Reply:
That's ok if default enabled plugins not influence the cluster bootstrap.